cmd/openshift-tests/images.go

```diff
@@ -9,14 +9,15 @@ import (
 	"strings"
 	"time"
 
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
```

Comment:
nit:order :trollface:

Reply:
also the package name is weird :-) `pkg/monitor/api` would be nicer ;-)

Reply:
> also the package name is weird :-) `pkg/monitor/api` would be nicer ;-)

`api` turned into a bad package name in kube and these aren't quite serializeable.  I'll compare against http and httptest: that moment they realized fully qualified names existed for a reason.